main.py

```diff
@@ -383,15 +384,17 @@ def onReadyReadStandardError(self):
 	def onXqemuStateChanged(self):
 		if self.inst.isRunning:
 			self.runButton.setText('Stop')
+			self.pauseButton.setEnabled(True)
 		else:
 			self.runButton.setText('Start')
+			self.pauseButton.setEnabled(False)
```

Comment:
nit: This could potentially be extended by looping over a list of UI elements which must be disabled.
I feel like we should also disable the restart / screenshot button and things like the "Edit &rarr; Settings" dialog option.